.github/workflows/build.yml

```diff
@@ -1,33 +1,44 @@
 name: Build & Deploy
+
+
 on:
   push:
     branches:
-      - master
```

Comment:
Just realized this should be master not main correct? @Zeroto521

Reply:
I think it better to let one branch manage to build and deploy, otherwise different versions would get conflict.

Reply:
https://github.com/Flow-Launcher/flow-launcher.github.io/commit/f86526fe3b54a4c7c2913ab5b4795897c42e78ae#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721

You are right, I forget that